packages/xod-client-electron/src/arduinoDependencies/messages.js

```diff
@@ -0,0 +1,40 @@
+import * as R from 'ramda';
+import { catMaybies } from 'xod-func-tools';
+import { getLibraryNameFromUrl } from 'xod-arduino-deploy';
+
+// =============================================================================
+//
+// Utils
+//
+// =============================================================================
+
+/**
+ * Returns a comma separated and wrapped with quotes library names.
+ * :: [String] -> String
+ */
+const getLibraryNames = R.pipe(
+  R.map(getLibraryNameFromUrl),
```

Comment:
No code in `messages.js`! A caller must provide args ready for dumb insertion/formatting. Otherwise localization will become harder for no reason.